src/main/resources/assets/numismatic-overhaul/lang/uk_ua.json

```diff
@@ -0,0 +1,69 @@
+{
+  "item.numismatic-overhaul.bronze_coin": "Бронзова монета",
+  "item.numismatic-overhaul.silver_coin": "Срібна монета",
+  "item.numismatic-overhaul.gold_coin": "Золота монета",
+  "item.numismatic-overhaul.money_bag": "Мішок із грішми",
+  "block.numismatic-overhaul.shop": "Крамниця",
+  "block.numismatic-overhaul.inexhaustible_shop": "Невичерпна крамниця",
+  "block.numismatic-overhaul.inexhaustible_shop.tooltip": "Лише в творчості — ніколи не вичерпуються запаси",
+  "block.numismatic-overhaul.piggy_bank": "Свинка-скарбничка",
+  "gui.numismatic-overhaul.shop.price": "Ціна",
+  "gui.numismatic-overhaul.shop.inventory_title": "Інвентар крамниці",
+  "gui.numismatic-overhaul.shop.merchant_title": "Крамниця",
+  "gui.numismatic-overhaul.shop.transfer_tooltip.enabled": [
+    "Hoppers",
+    {
+      "text": " can ",
+      "color": "#28FFBF"
+    },
+    "вставте \nпредметів у цю крамницю"
+  ],
+  "gui.numismatic-overhaul.shop.transfer_tooltip.disabled": [
+    "Hoppers",
+    {
+      "text": " cannot ",
+      "color": "#EB1D36"
+    },
+    "вставте \nпредметів у цю крамницю"
+  ],
+  "gui.numismatic-overhaul.purse_title": "Гаманець",
+  "numismatic-overhaul.empty": "Порожньо",
+  "filled_map.end_city": "Мапа дослідника Енду",
```

Comment:
is this what vanilla uses for maps sold by villagers?

Reply:
https://crowdin.com/translate/minecraft/10038/enus-uk#5360201